locale/ar/docs/guides/anatomy-of-an-http-transaction.md

```diff
@@ -399,5 +398,5 @@ http.createServer((request, response) => {
 [`express`]: https://www.npmjs.com/package/express
 [`router`]: https://www.npmjs.com/package/router
 [`pipe`]: https://nodejs.org/api/stream.html#stream_readable_pipe_destination_options
-[`Error` documentation]: https://nodejs.org/api/errors.html
+[`Error` مستندات]: https://nodejs.org/api/errors.html
```

Comment:
This also breaks the anchor link above. You need to make the same change, or leave this change, and translate the right square brackets, while copying the english into the left square brackets of the reference